frontend/src/app/components/settings/view/SettingsView.ts

```
@@ -28,7 +49,37 @@ export class SettingsView {
+            switch (setting) {
+                case 'replayInterval':
+                    return ConfigService.getConfigValueWithNoUserSettingsOverride('settings.replayInterval', 5);
+                case 'timezoneForEmails':
+                    return ConfigService.getConfigValueWithNoUserSettingsOverride(
+                        'settings.timezoneForEmails',
+                        'Europe/Paris'
+                    );
+                case 'email':
+                    if (this.isEmailFromUserInsteadOfSettings()) {
+                        return this.user.email;
+                    }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)